src/H.jsx

```diff
@@ -0,0 +1,4 @@
+export function H(props){
+    return <h1></h1>
```

Comment:
What's an 'H'. Don't use this kind of names.

What's the purpose of this component?